### PR TITLE
updated pull documentation

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[Flutter] Fix userProfile documentation.](https://trello.com/c/t6DBjZd5/1384-flutter-fix-userprofile-documentation)

File: lib/src/user_profile.dart [hunk 1]
```diff
@@ -50,8 +50,8 @@ abstract class UserProfile {
   /// [String value] - value to add to array
   Future<void> pushUnique(String key, String value);
 
-  /// Create array property, if property does not exist and remove value from array
-  /// You can only use it on array properties or properties that do not exist yet
+  /// If a custom property exists, its value is an array, and the specified value is present within that array, then this will remove the specified value from the array.
+  /// You can only use it on array properties that exist.
   /// [String key] - property name for array property
```
Comment (arifBurakDemiray) [inline lib/src/user_profile.dart:55]: It is a long sentence, I had to read 2-3 times to understand. Maybe sentence can be shortened. Like
`Removes a value from an array property if exists`
It is too short but you got the idea

**Review by arifBurakDemiray — APPROVED** [review]

File: lib/src/user_profile.dart (hunk 1 above)
Comment (ArtursKadikis) [reply lib/src/user_profile.dart:55]: the sentence is still to complicated, pls rewrite it

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

File: lib/src/user_profile.dart (hunk 1 above)
Comment (peterBrxwn) [reply lib/src/user_profile.dart:55]: fixed

**Pull request merged**